crates/cheatnet/src/forking/state.rs

```diff
@@ -97,10 +97,11 @@ impl StateReader for ForkStateReader {
             return Ok(cache_hit);
         }
 
-        match self.runtime.block_on(
+        match self.runtime.block_on(async {
             self.client
-                .get_class_hash_at(self.block_id, contract_address.to_field_element()),
-        ) {
+                .get_class_hash_at(self.block_id, contract_address.to_field_element())
+                .await
+        }) {
```

Comment:
This looks a bit weird to have block on and then async inside. Why was this added?

Reply:
good catch! I test sth and add it by mistake